nerdlets/nr1-cloud-migrator-nerdlet/components/complexity-profiler/index.js

```diff
@@ -232,7 +232,7 @@ export default class ComplexityProfile extends React.PureComponent {
               summary.languages = [...new Set(languages)];
               summary.guids = [...new Set(guids)];
 
-              console.debug("summary obj", summary);
+              console.debug('summary obj', summary);
```

Comment:
out of curiosity, why is this here?

Reply:
@caylahamann probably not for the user... I can remove it.  Maybe it was just a troubleshooting step someone forgot to remove?

Reply:
Actually this file has a series of console.log statement as well. so not knowing why, I'm thinking it's better to leave it in. Let me know what you think.